tools/azure-sdk-tools/devtools_testutils/proxy_testcase.py

```diff
@@ -57,11 +57,21 @@ def get_recording_id():
 def get_test_id():
     # type: () -> str
     # pytest sets the current running test in an environment variable
+    # the path to the test can depend on the environment, so we can't assume this is the path from the repo root
     setting_value = os.getenv("PYTEST_CURRENT_TEST")
-
     path_to_test = os.path.normpath(setting_value.split(" ")[0])
-    path_components = path_to_test.split(os.sep)
-
+    full_path_to_test = os.path.abspath(path_to_test)
```

Comment:
<snipped from problem description so I can compare to code base>

`sdk/search/azure-search-documents/tests/test_search_indexer_client_live.py::TestSearchIndexersClient::test_delete_indexer_if_unchanged (call)'`

When debugging the same test via VS Code, PYTEST_CURRENT_TEST is
`tests/test_search_indexer_client_live.py::TestSearchIndexersClient::test_delete_indexer_if_unchanged (call)`.